www/api/class/centreon_wiki.class.php

```diff
@@ -45,8 +45,11 @@ class CentreonWiki extends CentreonWebService
     public function postCheckConnection()
     {
         $sql_host = explode(':', $this->arguments['host']);
-        $host = $sql_host[0];
-        $port = isset($sql_host[1]) ? $sql_host[1] : '3306';
+
+        $host = filter_var($sql_host[0], FILTER_SANITIZE_STRING);
+        if (isset($sql_host[1])) {
```

Comment:
default value if $sql_host[1] not set?

Reply:
solved with [last commit](https://github.com/centreon/centreon/pull/8875/commits/af803bb411c090c8f142d334db066cf0cf4868a5)